### DISCUSSION TITLE
Estágio remoto

### DISCUSSION THREAD

**Discussion by schirrel** [frontendbr/forum#2003] [Discussão]
Boa tarde devs.

Estou mentorando/ajudando novos devs na minha cidade, porem a cidade é bem pequena e no interior do MS, e não não existem empresas de t.i. que oferecem estágio.
Aqui existem duas faculdades de T.I.: UFMS, a que me formei, e o IFMS. 
Cada uma das instituições tem parceria com apenas 1 empresa, que contrata no máximo 2 devs por ano, cada.
Eu mesmo, trabalhei como dev por quase 5 anos, mas muito longe daqui, voltei a pouco por motivos familiares, tanto que no momento nem estou trabalhando como dev (longa historia).
Sigo ajudando a galera, mentorando, ensinando, propondo atividades, tirando duvidas, fazendo tudo o que posso, mas a queixa principal gira sempre em torno de estágio. 
Até fico garimpando estágios  remotos para a galera, mas realmente é bem escaço. 
Imagino que essa seja a realidade de toda as cidades pequenas, sobre o mundo de T.I.
Gostaria de saber se vocês conhecem algo mais especifico pra estágios remotos da nossa área, programas ou algumas empresas que tem uma grande oferta de vagas, e  também estive pensando, da mesma maneira que o frontendbr usa sua visibilidade pra anunciar vagas efetivas, e se tivéssemos um frontendbr/estágios? Bem mais especifico.

Enfim, é isso galera, brigado pelo espaço :]

**Comment by Clara755** [comment]
Ei! 
Eu trabalho em uma empresa que contrata dev estágio home office!! Me manda um email que podemos trocar mais informações por lá! claradbmello@gmail.com
